packages/unit-jest/package.json

```diff
@@ -50,6 +50,8 @@
     "vue": "^3.2.41"
   },
   "peerDependencies": {
+    "@swc/core": "^1.3.19",
```

Comment:
Is it possible to add these as direct dependencies of the AE?
I set as dependencies the packages that the user isn't supposed to configure itself and from which he doesn't need to import anything or reference them explicitly in its code
Those packages are usually toolchain packages working behind the curtains to get a particular software work, as it is in this case

This increases the download size of the package, but avoids a lot of dependency problems when upgrading, and the mental burden of upgrading them altogether

What do you think?

Reply:
I think it depends on the likelihood of us wanting to use SWC capabilities broadly for typescript quasar. That might be more for you to answer. Since it is an experimental capability at this time, having it here until it matures seems to make sense.

Reply:
That makes sense, and I'd avoid to bloat the standard install version with more that it needs for now
We can just move them in later on

I don't think Quasar toolchain will move towards SWC in the near future anyway